rootdir/ueventd.shinano.rc

```diff
@@ -155,3 +155,7 @@
 /sys/devices/system/cpu/cpuquiet/rqbalance/balance_level             0660 system system
 /sys/devices/system/cpu/cpuquiet/rqbalance/nr_run_thresholds         0660 system system
 /sys/devices/system/cpu/cpuquiet/rqbalance/nr_down_run_thresholds    0660 system system
+
+# BT/FM/ANT+ V4L2
+/dev/brcm_bt_drv          0660   bluetooth  net_bt_stack
+/dev/radio0               0666   root       root
```

Comment:
Yeah, I saw it.
But we have the specific block (for BT FM) at the end of the stock ueventd. And yes it is also duplicate in stock file.